demo/src/app/components/+rating/rating-section.list.ts

```diff
@@ -43,6 +44,15 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/custom/custom'),
         html: require('!!raw-loader?lang=markup!./demos/custom/custom.html'),
         outlet: DemoRatingCustomComponent
+      },
+      {
+        title: 'Select on enter',
+        description: `Use this example for accessibility. After press <code>Enter</code>
```

Comment:
Key navigation example. Focus on rating and use arrow keys to set its value, then press `Enter` to select the value, after this, the rating state will be changed to readonly.